src/main/java/com/spotify/docker/client/messages/Event.java

```diff
@@ -91,7 +143,50 @@ public String toString() {
         .add("status", status)
         .add("id", id)
         .add("from", from)
+        .add("type", type)
+        .add("action", action)
+        .add("actor", actor)
         .add("time", time)
+        .add("timeNano", timeNano)
         .toString();
   }
+
+  public static class Actor {
+    @JsonProperty("ID") private String id;
+    @JsonProperty("Attributes") private ImmutableMap<String, Object> attributes;
```

Comment:
Could this just be `ImmutableMap<String, String>` or do we expect nested JSON objects?